### PR TITLE
📝 Adds blog post about starting with vim

### PR DESCRIPTION
**Pull request by cynektix**

### CODE REVIEW

File: _posts/2017-04-18-finally-switching-to-vim.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Before I started using vim, I thought about writing a blog article about switching to vim. I imagined to give some helpful resources for others to do the same. So when someone else in my company or any other friends wants to switch, I just give this person the link to this article and the person can start immediately.
+
+But then I realized the switch to vim is more like a journey. And a journey needs to be done by everyone who wants to switch.
+
+For example giving a person a vim configuration to work with is like starting a React app with a starter project that has a lot of boilerplate. Yes it works, but you don't know how. This way you get to an intermediate level really fast, but to an expert level maybe never. I think most of the people who use or who want to use vim do this because they want to get experts. So this post is about how to start wit vim, but not so much about configuration or plugins because this is very subjective in the end and everyone works a little bit different.
```
Comment (brainlessdeveloper) [inline _posts/2017-04-18-finally-switching-to-vim.md]: `because they want to become experts`

File: _posts/2017-04-18-finally-switching-to-vim.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+practical_dev_link:
+---
+
+First, a little history of my editor usage. 10 years ago I started with notepad and fastly switched to Notepad++. Then there came a time of Eclipse but I never loved working with it. A few years ago I got to know PhpStorm and RubyMine and really liked working with the awesome IDEs of JetBrains.
+
+***
+
+Two of my colleagues are already using vim and are super happy with it. So I decided I should give it a try, too. I knew it would be hard and that I had to plan with not being productive for at least the first two weeks.
+
+Before using vim I thought you can do every awesome thing with vim out of the box. Now I know better 🙃 Don't get me wrong here, vim is awesome without using any plugins already. But if you are used to a fully featured IDE you expect a lot of the features that made you productive. One example is fuzy file opening. Vim already is quite good at this out of the box, but with something like [fzf](https://github.com/junegunn/fzf) I like it a little bit more.
```
Comment (brainlessdeveloper) [inline _posts/2017-04-18-finally-switching-to-vim.md]: `fuzy` -> `fuzzy`

File: _posts/2017-04-18-finally-switching-to-vim.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* [fzf](https://github.com/junegunn/fzf)
+* [vim-surround](https://github.com/tpope/vim-surround)
+* [vim-gitgutter](https://github.com/airblade/vim-gitgutter)
+* [YouCompleteMe](https://github.com/Valloric/YouCompleteMe)
+
+And every programming language normally has a vim plugin on its own.
+
+***
+
+After one week I'm not as productive as with RubyMine. If I really need to get something done fast I still open RubyMine. But most of the time I use vim now and I enojoy working with it. Two more weeks and RubyMine stays closed. What I miss the most is the awesome multi cursor implementation of RubyMine. But I already have been told that in vim you do things differently 😉
```
Comment (brainlessdeveloper) [inline _posts/2017-04-18-finally-switching-to-vim.md]: `enjoy`

File: _posts/2017-04-18-finally-switching-to-vim.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+[This Github repository](https://github.com/mhinz/vim-galore) provides a long guide to vim.
+
+[Here are 68 free screencasts about vim](http://vimcasts.org/episodes/archive/) and [here are two more free ones and other paid ones](https://thoughtbot.com/upcase/onramp-to-vim).
+
+You also might wanna have a look at [neovim](https://neovim.io/).
+
+Vim has a very rich ecosystem of plugins. There are multiple ways of installing plugins. I started with [pathogen.vim](https://github.com/tpope/vim-pathogen) and it was ok. But my coworkers showed me the advantages of [vim-plug](https://github.com/junegunn/vim-plug) and as they are using vim-plug I switched, too.
+
+Then you have to decide which plugins to use (and not to use 😉) and how to customize your settings and keymaps. As a starting point I will list some plugins someone might have a look at. The most important thing here is: read through the documenation of each plugin and decide whether you need it or not!
```
Comment (tomhoule) [inline _posts/2017-04-18-finally-switching-to-vim.md]: `:s/documenation/documentation/`

File: _posts/2017-04-18-finally-switching-to-vim.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+* [vim-sensible](https://github.com/tpope/vim-sensible)
+* [fzf](https://github.com/junegunn/fzf)
+* [vim-surround](https://github.com/tpope/vim-surround)
+* [vim-gitgutter](https://github.com/airblade/vim-gitgutter)
+* [nerdtree](https://github.com/scrooloose/nerdtree)
+
+And every programming language normally has a vim plugin on its own.
+
+Finally, [start playing vimgolf](http://www.vimgolf.com/) from the beginning. It's never to early to start (I had that thought). Just look at the other solutions and try to understand what they did.
```
Comment (tomhoule) [inline _posts/2017-04-18-finally-switching-to-vim.md]: `:s/to early/too early/`

**Comment by cynektix** [comment]
:D nice review comments

**Pull request merged**